internal/backend/httpsvc/config.go

```diff
@@ -80,6 +80,7 @@ var Configs = configset.Set[Config]{
 			UnimportantRegexes: []string{
 				`^/autokitteh.+/(Get|List)$`, // gRPC Get and List methods
 				`/(healthz|readyz)$`,         // Kubernetes health checks
+				`^/oauth/|/oauth$|/save$`,    // Connection initialization
```

Comment:
shouldn't we see those in logs?

Reply:
Yes - with the debug level, it's redundant as info, considering all the other intercepted requests in the init flow

Reply:
I would even go further and say that some of the intercepted requests in this flow shouldn't be logged at all, so we don't leak sensitive user data, but that's a task for another time - see ENG-1249.